transforms/angle-brackets/__testfixtures__/reserved.output.hbs

```diff
@@ -0,0 +1,3 @@
+<SomeComponent @foo={{this.action}} />
```

Comment:
I don't think we can assume just to be able to add `this.` to these properties. What about the case where they are actually passed arguments that need `@propertyName` instead?

Reply:
Agreed now that I see this.

Reply:
@patocallaghan i think we should revisit this after the conversation in https://github.com/ember-codemods/ember-no-implicit-this-codemod/issues/11#issuecomment-513990337

Reply:
Sure, it was news to me 😄 FWIW I did implement a `dontAssumeThis` mode in this PR https://github.com/ember-codemods/ember-no-implicit-this-codemod/pull/12 but the more common use case is probably the `this.` one